package.json

```diff
@@ -53,7 +53,7 @@
     "babel-plugin-transform-es2015-modules-commonjs": "6.22.0",
     "babel-plugin-transform-object-rest-spread": "6.22.0",
     "babel-plugin-transform-runtime": "6.22.0",
-    "babel-preset-es2015": "6.22.0",
+    "babel-preset-latest": "6.22.0",
```

Comment:
It looks we can remove `transform-async-to-generator` which is included in this preset.